www-apps/grafana-bin/grafana-bin-5.2.3.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user systemd
+
+MY_PN=${PN/-bin/}
+S=${WORKDIR}/${MY_PN}-${PV}
+
+DESCRIPTION="Gorgeous metric viz, dashboards & editors for Graphite, InfluxDB & OpenTSDB"
+HOMEPAGE="https://grafana.org"
+SRC_URI="https://s3-us-west-2.amazonaws.com/grafana-releases/release/${MY_PN}-${PV}.linux-amd64.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+QA_EXECSTACK="usr/share/grafana/vendor/phantomjs/phantomjs"
+QA_PRESTRIPPED=${QA_EXECSTACK}
+
+pkg_setup() {
+	enewgroup grafana
+	enewuser grafana -1 -1 /usr/share/grafana grafana
+}
+
+src_install() {
+	keepdir /etc/grafana
```

Comment:
do you really need to keepdir the directory before installing into it? it wil not be empty in anyway.